### ISSUE TITLE
Menu panes are reopened unexpectedly

### ISSUE THREAD

**Issue by linusc-king** [defold/editor2-issues#269]
### Expected behaviour
Menu panes are not reopened unexpectedly.

### Actual behaviour
Menu pane is being reopened.

### Steps to reproduce
Click on a menu item so the menu pane is shown. Hover on the menu item text. You will see that the menu pane is flickering regularly as if it is being reopened.

<hr/>
Defold version: 1.2.95
Defold sha: 7ca47a7f7aa0e4809e31ab2274fd1693a1fc5a7e
Platform: Windows 7 6.1 (x86)
Java version: 1.8.0_102-b14

**Comment by RagnarSvenssonKing** [comment]
I investigated this a bit, it seems it is caused by new target devices being detected, which in turn rebuilds the menu system. We verified this by disconnecting the network, which stopped the flickering. Enabling the network (and restarting the target detection service) made it flicker again. It's not completely clear to me how this should be fixed, it might require some re-design of how the menus are updated.
